SU2_CFD/include/fluid/CFluidModel.hpp

```diff
@@ -119,10 +119,26 @@ class CFluidModel {
    */
   su2double GetCv() const { return Cv; }
 
+  /*!
+   * \brief Compute and return fluid mean molecular weight in kg/mol.
+   */
+  template <class Vector_t>
+  static su2double ComputeMeanMolecularWeight(const Vector_t& molar_masses, const su2double* val_scalars) {
+    su2double OneOverMeanMolecularWeight = 0.0;
+    su2double val_scalars_sum = 0.0;
+
+    for (size_t i_scalar = 0; i_scalar < molar_masses.size() - 1; i_scalar++) {
+      OneOverMeanMolecularWeight += val_scalars[i_scalar] / (molar_masses[i_scalar] / 1000);
+      val_scalars_sum += val_scalars[i_scalar];
+    }
+    OneOverMeanMolecularWeight += (1 - val_scalars_sum) / (molar_masses[molar_masses.size() - 1] / 1000);
+    return 1 / OneOverMeanMolecularWeight;
+  }
```

Comment:
`1 / OneOver...` is a bit unnecessary in my eyes
```suggestion
  /*!
   * \brief Compute and return fluid mean molecular weight in kg/mol.
   */
  template <class Vector_t>
  static su2double ComputeMeanMolecularWeight(const Vector_t& molar_masses, const su2double* val_scalars) {
    su2double MeanMolecularWeight = 0.0;
    su2double val_scalars_sum = 0.0;

    for (size_t i_scalar = 0; i_scalar < molar_masses.size() - 1; i_scalar++) {
      MeanMolecularWeight += molar_masses[i_scalar] / val_scalars[i_scalar];
      val_scalars_sum += val_scalars[i_scalar];
    }
    MeanMolecularWeight += molar_masses[molar_masses.size() - 1] / (1 - val_scalars_sum);
    /*--- Convert from [g/mol] of the individual molar masses to [kg/mol]. ---*/
    return MeanMolecularWeight / 1000;
  }
```

Reply:
You are indeed correct and I am stupid :facepalm: :+1:

Reply:
Thank you so much Tobias!! :)